docs/source/install.rst

```diff
@@ -143,13 +144,12 @@ Run the :code:`01_rgb_preview.py` example inside a Docker container on a Linux h
        luxonis/depthai-library:latest \
        python3 /depthai-python/examples/01_rgb_preview.py
 
-To allow the container to update X11 you may need to run :code:`xhost local:root` on
-the host.
+To allow the container to update X11 you may need to run :code:`xhost local:root` on the host.
 
-KVM
-***
+Kernel Virtual Machine
```

Comment:
```suggestion
Kernel Virtual Machine (KVM)
```
I'm mostly familiar with KVM shortcut, not the full qualified name, but it's a minor thing

Reply:
I'll change it next time if you don't mind @VanDavv. I'll be adding Virtual Box to the VM section.